src/mattack_actors.h

```diff
@@ -124,7 +124,10 @@ class gun_actor : public mattack_actor
         int max_ammo = INT_MAX; /** limited also by monster starting_ammo */
 
         /** Description of the attack being run */
-        std::string description = "The %s fires its %s";
+        std::string description = "The %1$s fires its %2$s!";
+#if 0
+        _("The %1$s fires its %2$s!"); // makes xgettext spot this string
+#endif
```

Comment:
That's an ugly (and fragile) way of ensuring. I'd suggest something like this:
```c++
if( !obj.read( "description", description ) && description.empty() ) {
  description = _( "The %1$s fires its %2$s!" );
}
```
in matack_actors.cpp.